AssemblerLib/MeshComponentMap.cpp

```diff
@@ -285,4 +285,44 @@ std::vector<GlobalIndexType> MeshComponentMap::getGlobalIndicesByComponent(
     return global_indices;
 }
 
+GlobalIndexType MeshComponentMap::getLocalIndex(
+    Location const& l,
+    std::size_t const comp_id,
+    std::size_t const range_begin,
+    std::size_t const range_end) const
+{
+    GlobalIndexType const global_index = getGlobalIndex(l, comp_id);
+#ifndef USE_PETSC
+    return global_index;
```

Comment:
TODO: add `(void) range_begin; (void) range_end` in order to silence compiler warnings.